### DISCUSSION TITLE
Bridge estimator An alternative to unit root tests: differentiate between nonstationary versus stationary models and select optimal lag

### DISCUSSION THREAD

**Discussion by revolutionneur** [alan-turing-institute/sktime#5599] [Ideas]
Hi all, 
I am new to time series field, I conducting some research about unit root tests and the needed orders of integration.
and I came a cross this article https://doi.org/10.1016/j.jspi.2012.10.009, with many related article too to Bridge and Lasso estimation of intgeration oredrs as a better estimators than unit root tests.

will these estimators be imlemented in sktime ,

**Comment by fkiraly** [comment]
> will these estimators be imlemented in sktime 

if someone implements them (or interfaces them), certainly!

`sktime` follows a "library" or "marketplace" paradigm of curation - you can add sth if you want, there is an extension template for parameter estimators: https://github.com/sktime/sktime/blob/main/extension_templates/param_est.py

Here's the general "implementing estimator" guide: https://www.sktime.net/en/latest/developer_guide/add_estimators.html
